run-publishing.sh

```diff
@@ -22,6 +22,7 @@ export DEV_ENVIRONMENT="Y"
 export IS_PUBLISHING="Y"
 export RELOAD_TEMPLATES="Y"
 export TEMPLATES_DIR=src/main/web/templates/handlebars
+export ENABLE_LOOP11="N"
```

Comment:
NO NO NO NO NO.

Reply:
Sorry. I appreciate you are following the existing style but this ☝️ is _terrible_. Please use _true_ or _false_